SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentTestUtils.h

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/EntityComponentTypes.h"
+
+namespace SpatialGDK
+{
+
+static const Schema_FieldId kEventId = 1;
+static const Schema_FieldId kEventIntFieldId = 2;
+
+static const Schema_FieldId kTestDoubleFieldId = 1;
+
+inline ComponentData CreateTestComponentData(Worker_ComponentId id, double value)
```

Comment:
Nit: I think unreal standards would have these args as const value. I am not really fussed though. 

Also, do we want capitals in variable names?

Reply:
You're right about capitalisation. I think non-const is fine though. Otherwise - could you share an example?

Reply:
Fixed capitalisation.

Reply:
https://docs.unrealengine.com/en-US/Programming/Development/CodingStandard/index.html

"Const should also be preferred on by-value function parameters and locals."

Reply:
Updated.